ui/snippets/networkMenu/NetworkLogo.tsx

```diff
@@ -50,39 +51,41 @@ const NetworkLogo = ({ isCollapsed, onClick }: Props) => {
   const iconStyle = useColorModeValue({}, !config.UI.sidebar.icon.dark ? darkModeFilter : {});
 
   return (
-    // TODO switch to <NextLink href={ href } passHref> when main page for network will be ready
-    <Box
-      as="a"
-      href={ route({ pathname: '/' }) }
-      width={{ base: 'auto', lg: isCollapsed === false ? '120px' : '30px', xl: isCollapsed ? '30px' : '120px' }}
-      height={{ base: '24px', lg: isCollapsed === false ? '24px' : '30px', xl: isCollapsed ? '30px' : '24px' }}
-      display="inline-flex"
-      overflow="hidden"
-      onClick={ onClick }
-      flexShrink={ 0 }
-      aria-label="Link to main page"
-    >
-      { /* big logo */ }
-      <Image
-        w="auto"
-        h="100%"
-        src={ logoSrc }
-        alt={ `${ config.chain.name } network logo` }
-        fallback={ <LogoFallback isCollapsed={ isCollapsed }/> }
-        display={{ base: 'block', lg: isCollapsed === false ? 'block' : 'none', xl: isCollapsed ? 'none' : 'block' }}
-        style={ logoStyle }
-      />
-      { /* small logo */ }
-      <Image
-        w="auto"
-        h="100%"
-        src={ iconSrc }
-        alt={ `${ config.chain.name } network logo` }
-        fallback={ <LogoFallback isCollapsed={ isCollapsed } isSmall/> }
-        display={{ base: 'none', lg: isCollapsed === false ? 'none' : 'block', xl: isCollapsed ? 'block' : 'none' }}
-        style={ iconStyle }
-      />
-    </Box>
+    <NextLink href={ '/' } passHref legacyBehavior>
```

Comment:
I think, this change is not related to the feature itself. I would leave it as it was, and clicking on the logo will hard-reload the page. So a user will be able to reset the app state if necessary.